### PR TITLE
delete products when goes out of stock

### PR DESCRIPTION
**Pull request by suraj-rathod**

### CODE REVIEW

File: includes/class-dropshipping-woocommerce-wpml-importer.php [hunk 1]
```diff
@@ -242,7 +243,115 @@ public function get_translation_id($sitepress,$product_ID){
...
+					$product_info = $this->product_information($productID);
+					
+					if(!empty($product_info)){
+						foreach($product_info as $pro_info){
+							$this->wc_deleteProduct($pro_info);  
+						}
+						if (!empty($_GET) && wp_verify_nonce($_GET['update_knawat_product_nonce'], 'update_knawat_product_action' ) ) {
+							$product_id = (int) $_GET['product_id'];
+							if (!empty($product_id)){
+								wp_redirect(admin_url('edit.php?post_type=product'));
```
Comment (iamdharmesh) [inline includes/class-dropshipping-woocommerce-wpml-importer.php:262]: Why we have redirect here?

**Comment by iamdharmesh** [comment]
@suraj-rathod help to resolve conflict please

File: includes/class-dropshipping-woocommerce-wpml-importer.php (hunk 1 above)
Comment (suraj-rathod) [reply includes/class-dropshipping-woocommerce-wpml-importer.php]: remove unnecessary redirection code in latest commit

**Comment by suraj-rathod** [comment]
> @suraj-rathod help to resolve conflict please
@iamdharmesh Please review it conflict is resolve

**Pull request merged**